csaf_2.0/examples/ROLIE/example-01-category.json

```diff
@@ -0,0 +1,10 @@
+{
+  "categories": [
+    {
+      "term": "Example Company Product A"
+    },
+    {
+      "term": "Example Company Product B"
+    }
+  ]
+}
```

Comment:
I think this also need to specify a scheme to which these categories relate. Though, I'm not sure what the value of that field should be, so:

```json
{
   "categories": {
      "category": [
         {
            "term": "Example Company Product A"
         },
         {
            "term": "Example Company Product B"
         }
      ],
      "scheme": "<reference to itself?>"
   }
}
```

Perhaps it can just be an arbitrary URN that gets then used in the individual documents. I think only terms specific to the `urn:ietf:params:rolie:category:information-type` category don't need to be explicitly noted in the category document.

For now, I'd recommend using a self-reference to the category document itself as the scheme.

Reply:
> I think this also need to specify a scheme to which these categories relate.

I agree that there SHOULD be a category scheme. However, while reading [RFC 5023 Section 7](https://datatracker.ietf.org/doc/html/rfc5023#section-7) I did not see that it is mandatory
```
    appInlineCategories =
       element app:categories {
           attribute fixed { "yes" | "no" }?,
           attribute scheme { atomURI }?,
           (atomCategory*,
           undefinedContent)
       }
```

Therefore, I skipped it. But I might have overlooked something. Can you please double-check that?

If it is necessary, I second your approach to app the reference to itself in the document.

Reply:
Nevertheless, you are right about mentioning of `category`.

Reply:
I fixed that in f44b6cb.

Reply:
You're right, it is optional. Let's leave it out. Thanks for the `category` fix!

Reply:
No worries. I also fixed that in the service document examples and fixed a filename error.